### PR TITLE
support scdb api with openapi

### PR DESCRIPTION
**Pull request by jingshi-ant**

### CODE REVIEW

File: api/scdb_api.proto [hunk 1]
```diff
@@ -72,7 +96,7 @@ message SCDBQueryRequest {
   string biz_request_id = 5;
 
   // Current database name
-  string db_name = 6;
+  string db_name = 6 [(google.api.field_behavior) = REQUIRED];
```
Comment (CodeWilliamHuang) [inline api/scdb_api.proto:99]: db_name should be optional?

Comment (tongke6) [reply api/scdb_api.proto:99]: +1

File: docs/api/scdb_api.swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+      scdb_session_id:
+        type: string
+        title: Scdb session id
+  scql.pb.SQLWarning:
+    type: object
+    properties:
+      reason:
+        type: string
+        title: Description of the warning
+    title: 'TODO: move SQLWarning to a proper place'
```
Comment (tyrone-yu) [inline docs/api/scdb_api.swagger.yaml:393]: 这个 title 是不是怪怪的？

File: api/scdb_api.proto (hunk 1 above)
Comment (jingshi-ant) [reply api/scdb_api.proto]: fixed

File: docs/api/scdb_api.swagger.yaml (hunk 2 above)
Comment (jingshi-ant) [reply docs/api/scdb_api.swagger.yaml]: fixed

**Review by tyrone-yu — APPROVED** [review]
LGTM

**Pull request merged**